index.test.js

```diff
@@ -489,4 +489,21 @@ describe('Render task definition', () => {
             }, null, 2)
         );
     });
+
+    test('error returned for invalid command', async () => {
+        core.getInput = jest
+            .fn()
+            .mockReturnValueOnce('task-definition.json')
+            .mockReturnValueOnce('web')
+            .mockReturnValueOnce('nginx:latest')
+            .mockReturnValueOnce('EXAMPLE=here')
+            .mockReturnValueOnce('awslogs')
+            .mockReturnValueOnce('awslogs-create-group=true\nawslogs-group=/ecs/web\nawslogs-region=us-east-1\nawslogs-stream-prefix=ecs')
+            .mockReturnValueOnce('key1=value1\nkey2=value2')
+            .mockReturnValueOnce('npm start --nice --please');
+
+        await run();
+
+        expect(core.setFailed).toBeCalledWith('Invalid command: \'npm start --nice --please\'. command must be a valid JSON array.');
```

Comment:
Please feel free to add additional insights to this, but seems like this change would render previously "valid" commands as "invalid" for other users. 

For example `npm start --nice --please` would be expected to pass earlier but with this change, users would be required to change the format in which they provide the `command` field. 

Is this correct?